packages/components/src/theme/types.ts

```diff
@@ -14,6 +14,46 @@ export type ThemeProps = {
 	 * `var(--my-custom-property)`) are _not_ supported values for this property.
 	 */
 	accent?: string;
+	/**
+	 * The background color.
+	 *
+	 * If a component explicitly has a background, it will be use this color.
```

Comment:
Nit/Typo:
> it will be use this color

This looks like the thing I do **all the time** where I change my mind about some wording but leave a bit of the old in with the new. Or maybe you just want a `/` in between `be` and `use` 🙂